### ISSUE TITLE
explain/fix why certain blocks don't work in blockbuilder

### ISSUE THREAD

**Issue by stelt** [enjalot/blockbuilder#125]
e.g. http://blockbuilder.org/mbostock/4063570

**Comment by enjalot** [comment]
yes, this is because we don't support relative paths to other blocks. I think the solution is to point this out explicitly (we should be able to recognize this error) and explain why its bad. The main problem with it is that if you copy paste the code into your own project the relative links will also break (they are magic specific to bl.ocks.org). 
We could recognize the error this throws and highlight it in the code editor with an explanation.

**Comment by enjalot** [comment]
this one is broken because of a relative path to an svg file used in an svg image tag
http://blockbuilder.org/mpmckenna8/f0abe7ae10eef35972bd

**Comment by enjalot** [comment]
this one is broken because of way i'm hacking jquery and <script> tags
http://blockbuilder.org/mbostock/5440492

**Comment by biovisualize** [comment]
The latest problem can be worked around by adding a dummy script tag before the vertex shader one. http://blockbuilder.org/biovisualize/1e46f0d03edd656b22fa
